unit_tests/EnumUnitTests.d

```diff
@@ -248,6 +248,64 @@ D", options);
 
 }
 
+// Test aliasing of all enum members in nested scope
+unittest
+{
+    Options options;
+    options.aliasEnumMembers = true;
+
+    assertTranslates(
+q"C
+enum Enum
+{
+    FOO,
+};
+
+struct Foo
+{
+    enum FooEnum
+    {
+        BAR,
+        BAZ,
+    };
```

Comment:
Compiling this gives me a warning `warning: declaration does not declare anything`. You need to add a field.